### PR TITLE
skip DevicePluginProbe in node serial presubmits

### PR DESCRIPTION
**Pull request by wgahnagl**
fixes https://github.com/kubernetes/test-infra/issues/24557

### CODE REVIEW

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 1]
```diff
@@ -484,7 +484,7 @@ presubmits:
         - --node-test-args=LocalStorageCapacityIsolation=true --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
         - --node-tests=true
         - --provider=gce
-        - --test_args=--nodes=1 --focus="\[Serial\]" --skip="\[Flaky\]|\[Benchmark\]|\[NodeSpecialFeature:.+\]|\[NodeSpecialFeature\]|\[NodeAlphaFeature:.+\]|\[NodeAlphaFeature\]|\[NodeFeature:Eviction\]|\[NodeFeature:DynamicKubeletConfig\]|\[Feature:Docker\]"
+        - --test_args=--nodes=1 --focus="\[Serial\]" --skip="\[Flaky\]|\[Benchmark\]|\[NodeSpecialFeature:.+\]|\[NodeSpecialFeature\]|\[NodeAlphaFeature:.+\]|\[NodeAlphaFeature\]|\[NodeFeature:Eviction\]|\[NodeFeature:DynamicKubeletConfig\]|\[Feature:Docker\]|\[NodeFeature:DevicePluginProbe\]"
```
Comment (ehashman) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:487]: Can you please also skip this on the containerd and cri-o presubmit versions of the job?

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 2]
```diff
@@ -484,7 +484,7 @@ presubmits:
         - --node-test-args=LocalStorageCapacityIsolation=true --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/"
         - --node-tests=true
         - --provider=gce
```
Comment (SergeyKanzhelev) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:486]: Can you please add a comment explaining why `NodeFeature:DevicePluginProbe` is skipped and potentially a TODO item how to get it back.

**Comment by SergeyKanzhelev** [comment]
Let's keep it temporarily since @fromanirh will be looking at the test failure.

/hold

**Comment by fromanirh** [comment]
/hold cancel
unfortunately it will took me more time than expected to dive deep enough in the test failure, so not worth holding too long

**Comment by SergeyKanzhelev** [comment]
@wgahnagl can you please address the comments?

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml [hunk 3]
```diff
@@ -696,7 +696,8 @@ presubmits:
           - '--node-test-args=--feature-gates=LocalStorageCapacityIsolation=true --container-runtime=remote --container-runtime-endpoint=unix:///run/containerd/containerd.sock --container-runtime-process-name=/usr/bin/containerd --container-runtime-pid-file= --kubelet-flags="--cgroups-per-qos=true --cgroup-root=/ --runtime-cgroups=/system.slice/containerd.service" --extra-log="{\"name\": \"containerd.log\", \"journalctl\": [\"-u\", \"containerd*\"]}"'
           - --node-tests=true
           - --provider=gce
-          - --test_args=--nodes=1 --focus="\[Serial\]" --skip="\[Flaky\]|\[Benchmark\]|\[NodeSpecialFeature:.+\]|\[NodeSpecialFeature\]|\[NodeAlphaFeature:.+\]|\[NodeAlphaFeature\]|\[NodeFeature:Eviction\]"
+          #TODO: NodeFeature:DevicePluginProbe is being skipped so it can be readded in 1.23 and avoid backporting changes
```
Comment (SergeyKanzhelev) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:699]: 1.23 is already shipped. Do you mean 1.24?

Comment (SergeyKanzhelev) [reply config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:699]: Maybe just say:

// NodeFeature:DevicePluginProbe tests are failing and being investigated in https://github.com/kubernetes/kubernetes/issues/106635

File: config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml (hunk 1 above)
Comment (SergeyKanzhelev) [inline config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml:487]: missed adding comment here

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

thank you!

**Pull request merged**